### PR TITLE
Feat/select notification channel

### PR DESCRIPTION
**Pull request by SEQUOIIA**

### CODE REVIEW

File: fake_dependencies/capability-service/capability-data.json [hunk 1]
```diff
@@ -79,5 +79,20 @@
...
+    "members": [
+      {
+        "email": "highlander@me.com"
+      },
+      {
+        "email": "jdog@me.com"
+      }
+    ],
+    "contexts": [],
+    "RootId": ""
```
Comment (ZaradarDFDS) [inline fake_dependencies/capability-service/capability-data.json]: should RootId, be rootId?

File: fake_dependencies/harald/server.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+app.get("/api/v1/channels", (req, res) => {
+    readFile("./channels-data.json")
+        .then(data => res.send(data));
+});
+
+
+// The API contract is in a state of flux. This endpoint may be removed at some point.
+// This endpoint is similar to DELETE /api/v1/connections
+app.post("/api/v1/channel/leave", (req, res) => {
```
Comment (ZaradarDFDS) [inline fake_dependencies/harald/server.js]: This endpoint is outdated, delete the code if not needed

File: fake_dependencies/harald/server.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+        })
+        .catch(err => {
+            console.log("Error: " + err);
+            res.status(500).json(err);
+        });
+});
+
+// The API contract is in a state of flux. This endpoint may be removed at some point.
+// This endpoint is similar to POST /api/v1/connections
+app.post("/api/v1/channel/join", (req, res) => {
```
Comment (ZaradarDFDS) [inline fake_dependencies/harald/server.js]: This endpoint is outdated, delete the code if not needed

File: src/Blaster.WebApi/Features/Channels/ChannelApiController.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
+using System;
```
Comment (ZaradarDFDS) [inline src/Blaster.WebApi/Features/Channels/ChannelApiController.cs]: Delete Join + Leave actions.

File: fake_dependencies/capability-service/capability-data.json (hunk 1 above)
Comment (SEQUOIIA) [reply fake_dependencies/capability-service/capability-data.json]: Seems to be RootId throughout the entirety of capability-data.json. RootId is nonetheless on the way out(AFAIK, feel free to correct me).

File: fake_dependencies/harald/server.js (hunk 2 above)
Comment (SEQUOIIA) [reply fake_dependencies/harald/server.js]: On it!

File: fake_dependencies/harald/server.js (hunk 3 above)
Comment (SEQUOIIA) [reply fake_dependencies/harald/server.js]: On it!

File: src/Blaster.WebApi/Features/Channels/ChannelApiController.cs (hunk 4 above)
Comment (SEQUOIIA) [reply src/Blaster.WebApi/Features/Channels/ChannelApiController.cs]: On it!

**Pull request merged**